p3/tests/test_models.py

```diff
@@ -43,6 +43,23 @@ def test_profile(self):
 
         self.assertEqual(attendee_profile, profile)
 
+    def test_assigned_to_email_is_saved_as_lowercase(self):
```

Comment:
I'd like to move this test somewhere inside `tests/test_tickets.py` (ie. global test/ directory, not the per-app directory)

Reply:
Please apologise the very late reply. I moved the test to `tests/test_tickets.py`.